karabo_data/tests/make_examples.py

```diff
@@ -140,6 +140,7 @@ def make_train_ids(path):
 from .mockdata.xgm import XGM
 from .mockdata.gec_camera import GECCamera
 from .mockdata.sidemic_camera import SidemicCamera
+from .mockdata.basler_camera import BaslerCamera
```

Comment:
The component called `SidemicCamera` is actually a basler camera. You could maybe then remove the other one. And since there are various Basler reference with difference resolution it would be also nice if this was configurable

Reply:
Sorry about the misnaming there.

Reply:
No worries, I probably did not know myself back then :)

Reply:
But also that raises again the point that it is impossible to know what hardware is hidden behind a name in files unless one has expert knowledge of the instrument setup...
I believe that the karabo device class of each device monitored will be stored in the new file format (under the `RUN` group) which might be enough most of the time

Reply:
Oh yes, that would be good. If it is, we should ensure there's a way to get it in karabo_data.

Reply:
For now, I've replaced the ```SidemicCamera``` with the ```BaslerCamera``` class